package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Actions Http Client",
   "main": "index.js",
   "scripts": {
-    "build": "rm -Rf ./_out && tsc && cp package*.json ./_out && cp *.md ./_out && cp LICENSE ./_out && cp actions.png ./_out",
```

Comment:
we need this to build a fully realized package that we're going to publish in _out.  The alternative is a make.js which is where I started with (and why shelljs/make was added.  that's an xplat outer build solution.

Reply:
Reverted this change, I was thinking that there would separate `build` and `release`/`publish` scripts